.github/workflows/rust.yml

```diff
@@ -53,6 +53,19 @@ jobs:
       # Iterate through all subcrates to ensure each compiles indpendently.
       - run: for d in $(for toml in $(find . -name Cargo.toml) ; do echo ${toml%/*} ; done | sort -r ) ; do echo "# $d" ; (cd $d ; cargo check --all-targets) ; done
 
+  build-fuzz:
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    container:
+      image: docker://rust:1.49.0-buster
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup toolchain add nightly
+      - run: cargo install cargo-fuzz
+      # Iterate through all fuzz crates to ensure each compiles indpendently.
+      - run: for d in $(find . -name fuzz | sort -r) ; do echo "# $d" ; (cd $d ; cargo +nightly fuzz build) ; done
```

Comment:
does this need `--features fuzzing` to work?

Reply:
On closer inspection, we don't actually need this feature at all. I've removed the features and we now rely on `cfg(fuzzing)` gates instead (which also prevents fuzzing deps from being pulled in when running `cargo check`

Reply:
Great --- I wasn't entirely sure why there was both a feature flag and a cfg!